src/torchmetrics/utilities/imports.py

```diff
@@ -12,57 +12,55 @@
-_PYSTOI_AVAILABLE: bool = package_available("pystoi")
-_FAST_BSS_EVAL_AVAILABLE: bool = package_available("fast_bss_eval")
-_MATPLOTLIB_AVAILABLE: bool = package_available("matplotlib")
-_SCIENCEPLOT_AVAILABLE: bool = package_available("scienceplots")
-_MULTIPROCESSING_AVAILABLE: bool = package_available("multiprocessing")
-_XLA_AVAILABLE: bool = package_available("torch_xla")
-_PIQ_GREATER_EQUAL_0_8: Optional[bool] = compare_version("piq", operator.ge, "0.8.0")
-_FASTER_COCO_EVAL_AVAILABLE: bool = package_available("faster_coco_eval")
-_MECAB_AVAILABLE: bool = package_available("MeCab")
-_MECAB_KO_AVAILABLE: bool = package_available("mecab_ko")
-_MECAB_KO_DIC_AVAILABLE: bool = package_available("mecab_ko_dic")
-_IPADIC_AVAILABLE: bool = package_available("ipadic")
-_SENTENCEPIECE_AVAILABLE: bool = package_available("sentencepiece")
+_JIWER_AVAILABLE = RequirementCache("jiwer")
+_NLTK_AVAILABLE = RequirementCache("nltk")
+_ROUGE_SCORE_AVAILABLE = RequirementCache("rouge_score")
+_BERTSCORE_AVAILABLE = RequirementCache("bert_score")
+_SCIPY_AVAILABLE = RequirementCache("scipy")
+_SCIPY_GREATER_EQUAL_1_8 = RequirementCache("scipy>=1.8.0")
+_TORCH_FIDELITY_AVAILABLE = RequirementCache("torch_fidelity")
+_LPIPS_AVAILABLE = RequirementCache("lpips")
+_PYCOCOTOOLS_AVAILABLE = RequirementCache("pycocotools")
+_TORCHVISION_AVAILABLE = RequirementCache("torchvision")
+_TORCHVISION_GREATER_EQUAL_0_8 = RequirementCache("torchvision>=0.8.0")
+_TORCHVISION_GREATER_EQUAL_0_13 = RequirementCache("torchvision>=0.13.0")
+_TQDM_AVAILABLE = RequirementCache("tqdm")
+_TRANSFORMERS_AVAILABLE = RequirementCache("transformers")
+_TRANSFORMERS_GREATER_EQUAL_4_4 = RequirementCache("transformers>=4.4.0")
+_TRANSFORMERS_GREATER_EQUAL_4_10 = RequirementCache("transformers>=4.10.0")
+_PESQ_AVAILABLE = RequirementCache("pesq")
+_GAMMATONE_AVAILABLE = RequirementCache("gammatone")
+_TORCHAUDIO_AVAILABLE = RequirementCache("torchaudio")
+_TORCHAUDIO_GREATER_EQUAL_0_10 = RequirementCache("torchaudio>=0.10.0")
+_SACREBLEU_AVAILABLE = RequirementCache("sacrebleu")
+_REGEX_AVAILABLE = RequirementCache("regex")
+_PYSTOI_AVAILABLE = RequirementCache("pystoi")
+_FAST_BSS_EVAL_AVAILABLE = RequirementCache("fast_bss_eval")
+_MATPLOTLIB_AVAILABLE = RequirementCache("matplotlib")
+_SCIENCEPLOT_AVAILABLE = RequirementCache("scienceplots")
+_MULTIPROCESSING_AVAILABLE = RequirementCache("multiprocessing")
+_XLA_AVAILABLE = RequirementCache("torch_xla")
+_PIQ_GREATER_EQUAL_0_8 = RequirementCache("piq>=0.8.0")
+_FASTER_COCO_EVAL_AVAILABLE = RequirementCache("faster_coco_eval")
+_MECAB_AVAILABLE = RequirementCache("MeCab")
+_MECAB_KO_AVAILABLE = RequirementCache("mecab_ko")
+_MECAB_KO_DIC_AVAILABLE = RequirementCache("mecab_ko_dic")
+_IPADIC_AVAILABLE = RequirementCache("ipadic")
+_SENTENCEPIECE_AVAILABLE = RequirementCache("sentencepiece")
 
-_LATEX_AVAILABLE: bool = shutil.which("latex") is not None
+_LATEX_AVAILABLE = shutil.which("latex") is not None
```

Comment:
```suggestion
_LATEX_AVAILABLE: bool = shutil.which("latex") is not None
```
Any reason for removing type here?

Reply:
It is not needed